test/run-drun/function-upgrades.drun

```diff
@@ -0,0 +1,8 @@
+install $ID function-upgrades/version0.mo ""
```

Comment:
Good test, though it took me  while to figure out what is  legal here ;-> Upgrade to 2 3 and 4 should fail, I think.